CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css

```diff
@@ -4,16 +4,9 @@ body {
     font-family: "Signika", sans-serif;
     font-weight: 500;
     letter-spacing: 1px;
-    background-image: url("../Images/img1.jpg");
-    height: 100vh;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-size: cover;
-    background-position: center;
 }
 
-.main {
+#main {
```

Comment:
Another ID? Why?

Reply:
No, this is the previous id. I've replaced class on id due to incorrect display in style ForgotAndResetPage and in other Views for change password

Reply:
This is not the right solution, use classes